cmake/process_options.cmake

```diff
@@ -171,3 +150,9 @@ if(ENABLE_KOKKOS)
 else()
     message(STATUS "ENABLE_KOKKOS is OFF.")
 endif()
+
+if (UNIX AND (${CMAKE_SYSTEM_PROCESSOR} MATCHES "(AMD64)|(X64)|(x64)|(x86_64)"))
```

Comment:
Does AMD64 support AVX-512?

Reply:
AMD does not support AVX512, but AMD64 is just one of the possible names of x86-64. It is indeed confusing :smile: